front/src/actions/message.js

```diff
@@ -74,15 +79,16 @@ function createActions(store) {
       });
       const messageText = state.currentMessageTextInput;
       try {
+        const id = uuid.v4();
         const newMessage = {
           text: messageText,
-          created_at: new Date()
+          created_at: new Date(),
+          id
         };
-        const tempId = uuid.v4();
         // we first push the message
         const newState = update(state, {
           messages: {
-            $push: [Object.assign({}, newMessage, { tempId })]
+            $push: [Object.assign({}, newMessage, { id })]
```

Comment:
why do you do that? the id is already in newMessage

Reply:
Modified to use find()